src/Maestro/Maestro.Data/Migrations/20240202111944_CodeEnabledSubscriptions.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Maestro.Data.Migrations
+{
+    public partial class CodeEnabledSubscriptions : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<bool>(
+                name: "SourceEnabled",
+                table: "Subscriptions",
+                type: "bit",
+                nullable: false,
+                defaultValue: false);
+
+            migrationBuilder.CreateTable(
+                name: "AssetFilters",
+                columns: table => new
+                {
+                    Id = table.Column<int>(type: "int", nullable: false)
+                        .Annotation("SqlServer:Identity", "1, 1"),
+                    Filter = table.Column<string>(type: "nvarchar(max)", nullable: true),
+                    SubscriptionId = table.Column<Guid>(type: "uniqueidentifier", nullable: true)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_AssetFilters", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_AssetFilter_Subscriptions_SubscriptionId",
+                        column: x => x.SubscriptionId,
+                        principalTable: "Subscriptions",
+                        principalColumn: "Id");
+                });
+
+            migrationBuilder.CreateIndex(
+                name: "IX_AssetFilter_SubscriptionId",
+                table: "AssetFilters",
+                column: "SubscriptionId");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
+                name: "AssetFilters");
```

Comment:
Please extract the name to const string so it's clear that we are dropping the same table as we are creating.

Reply:
@tkapin these are auto-generated by EF